app/javascript/components/search/controls/download/DownloadSelectionModal.js

```diff
@@ -118,6 +118,9 @@ export default function DownloadSelectionModal({ studyAccessions, show, setShow
             { showAzulSelectionPane &&
               <div>
                 <h4>Human Cell Atlas studies</h4>
+                <span className="detail">Available under
+                  the <a href="https://data.humancellatlas.org/about/data-use-agreement" target="_blank" rel="noopener noreferrer">HCA Data Use Agreement</a>
```

Comment:
Let's omit `rel="noopener noreferrer"`.  It was a security measure for browsers that are no longer supported.

[`target=_blank` has implied `rel=noopener` by default](https://chromestatus.com/feature/6140064063029248) since Chrome 88, Firefox 79, and Safari 12.  The last browser version without this feature was EOL'd last year.  

I imagine IE 11 isn't up to par here, and although it's not EOL'd upstream until June 15, 2022, it's already not supported on Terra, so we're not obliged to support it either.  [< 0.01%](https://mixpanel.com/s/2OIWfe) of SCP users are on Internet Explorer.

`noreferrer` was a security patch for poor `noopener` support, so these points mean it's also unnecessary.

```suggestion
                  the <a href="https://data.humancellatlas.org/about/data-use-agreement" target="_blank">HCA Data Use Agreement</a>
```

Reply:
agree with the analysis.  I'll leave this in for now because it's part of our eslint (my IDE added it automatically), and we can make a separate PR to strip these from everywhere

Reply:
Good idea, I've opened SCP-4133.